rmapi/upload.go

```diff
@@ -96,11 +98,12 @@ func (c *Client) Upload(ctx context.Context, args UploadArgs) error {
 	}
 
 	payload := ItemInfo{
-		ID:      args.ID,
-		Name:    args.Title,
-		Type:    "DocumentType",
-		Version: 1,
-		Parent:  args.ParentID,
+		ID:           args.ID,
+		Name:         args.Title,
+		Type:         "DocumentType",
+		Version:      1,
+		LastModified: strconv.FormatInt(time.Now().UnixNano()/1e6, 10),
```

Comment:
1. After we changed the type to int64, we no - longer need to do `strconv.FormatInt` here
2. From the code I think the expected value is milliseconds since epoch, right? I'd suggest just use `time.Now().Unix() * 1e3` (which loses sub-second precision but I'd argue that they are not really important here), or if you really want to keep the sub-second precision, use that plus `time.Now().Nanosecond() / 1e6`. The reason is that `UnixNano()` only works for a much shorter time range (see [its doc](https://pkg.go.dev/time#Time.UnixNano) for an explanation, not that this program is likely to be running after year 2262 but still).

Reply:
Also according to https://github.com/splitbrain/ReMarkableAPI/wiki/Storage#general-item-info it's supposed to be `ModifiedClient` and use RFC3339 string instead of unix timestamp.